.github/workflows/build.yml

```
@@ -239,7 +239,11 @@ jobs:
       - uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
       - uses: jupyterlab/maintainer-tools/.github/actions/check-links@v1
         with:
-          ignore_links: 'lite/ https://www.npmjs.com.* ^/.*'
+          ignore_links: |
```

```suggestion
          ignore_links: >-
```

I think this would fix the issue. `|` will keep the newlines, `>` won't! And `-` will trim the trailing newline. YAML multiline strings are such a pain in the ass.